ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestExecutorUtil.java

```diff
@@ -282,14 +286,16 @@ private static void initializeTopics(
 
       topic.getKeySchema().ifPresent(schema -> {
         try {
-          srClient.register(KsqlConstants.getSRSubject(topic.getName(), true), schema);
+          int id = srClient.register(KsqlConstants.getSRSubject(topic.getName(), true), schema);
+          System.out.print("h");
```

Comment:
where is the `id` use? What about the print call?

Reply:
Will remove